models/OrderDepositTransaction.go

```diff
@@ -0,0 +1,74 @@
+package models
+
+import (
+	"github.com/Sirupsen/logrus"
+	"github.com/delta/dalal-street-server/utils"
+)
+
+type OrderDepositTransaction struct {
+	TransactionId uint32 `gorm:"column:transactionId;not null" json:"transaction_id"`
+	OrderId       uint32 `gorm:"column:orderId;not null" json:"bid_id"`
+	IsAsk         bool   `gorm:"column:isAsk;not null" json:"is_ask"`
+	CreatedAt     string `gorm:"column:createdAt;not null" json:"created_at"`
+}
+
+func (OrderDepositTransaction) TableName() string {
+	return "OrderDepositTransactions"
+}
+
+// GetOrderDepositTransactionRef returns a reference of
+func GetOrderDepositTransactionRef(transactionID, orderID uint32, isAsk bool) *OrderDepositTransaction {
```

Comment:
can you call this makeOrderDepositeTxRef or something? `Get` doesn't seem to fit here. And does this need to be exported?

Reply:
Sorry about the comment xD
And yes, I'm using this User.go Line 2093

Reply:
Fixed this :+1: